python/lsst/pipe/drivers/skyCorrection.py

```diff
@@ -175,6 +187,15 @@ def loadImage(self, cache, dataId):
         bgOld = cache.butler.get("calexpBackground", dataId, immediate=True)
         image = cache.exposure.getMaskedImage()
 
+        if self.config.doDetection:
+            # We deliberately use the specified 'detectSigma' instead of the PSF, in order to better pick up
+            # the faint wings of objects.
+            results = self.detection.detectFootprints(cache.exposure, doSmooth=True,
+                                                      sigma=self.config.detectSigma, clearMask=True)
+            if hasattr(results, "background") and results.background:
+                # Restore any background that was removed during detection
```

Comment:
Just checking that this is just in case `detection.reEstimateBackground` was set to True by the user?    With defaults, results.background is None?

Reply:
Yes, that's exactly the intention: just being defensive.